spec/models/video_spec.rb

```diff
@@ -0,0 +1,68 @@
+require 'spec_helper'
+
+describe Video do
+  it { should validate_presence_of(:title) }
+  it { should validate_presence_of(:description) }
+  it { should belong_to(:category) }
+
+  describe ".search_by_title" do
+
+    it "should return an empty array if the search is an empty string" do

```

Comment:
I would recommend avoiding the word "should" inside the example message - read more about [here](http://www.betterspecs.org/#should)